packages/cli/src/command.js

```diff
@@ -35,6 +36,12 @@ export default class NuxtCommand extends Hookable {
   }
 
   async run () {
+    await this.callHook('setup', {
```

Comment:
I hope no one complains about hook name 😆 It is inside run so could be called like `run:before` but with logical perspective, it is really a setup step and internal hook so looks good and also not breaking users.